cli/tests/integration_tests.rs

```diff
@@ -694,16 +694,152 @@ fn ts_reload() {
 
 #[test]
 fn test_ast() {
-  let mod1 = util::root_path().join("cli/tests/subdir/mod1.ts");
+  let mod1 = util::root_path().join("cli/tests/subdir/print_hello.ts");
   assert!(mod1.is_file());
-  let mut deno = util::deno_cmd()
+  let output = util::deno_cmd()
     .current_dir(util::root_path())
     .arg("ast")
```

Comment:
Why not use `itest` instead? I would be much shorter?

Reply:
Right, will do that.